tests/locations/test_signals.py

```diff
@@ -0,0 +1,52 @@
+import datetime
+import json
+from unittest import mock
+
+import pytest
+from django.conf import settings
+from django.utils import timezone
+from locations import models
+from locations import signals
+
+
+@pytest.mark.django_db
+@mock.patch("locations.signals._notify_administrators")
+def test_report_failed_fixity_check(_notify_administrators: mock.Mock):
+    package = models.Package.objects.create(
+        current_location=models.Location.objects.create(
+            space=models.Space.objects.create()
+        )
+    )
+    sender = None
+    kwargs = {
+        "uuid": str(package.uuid),
+        "location": "/tmp",
+        "report": json.dumps(
+            {
+                "success": False,
+                "message": "Incorrect package checksum",
+                "failures": {"files": {"missing": [], "changed": [], "untracked": []}},
+                "timestamp": None,
+            }
+        ),
+    }
+    settings.TIME_ZONE = "America/Los_Angeles"
+    expected_time = timezone.make_aware(
+        datetime.datetime(2024, 5, 17, 11, 00),
+        timezone=timezone.activate(settings.TIME_ZONE),
```

Comment:
I have not tested but I think Django uses the `TIME_ZONE` setting by default in `make_aware`, so you could remove this `timezone` keyword argument. That could also make the `settings.TIME_ZONE` assignment above unnecessary.

Reply:
Is the `settings.TIME_ZONE = "America/Los_Angeles"` line still necessary?

Reply:
Not really. Thanks, let me remove it.